### ISSUE TITLE
Add Gradient Boosting Machines algorithm and Neural Network

### ISSUE THREAD

**Issue by 4ryn** [BamaCharanChhandogi/Diabetes-Prediction#89]
The use  of Gradient Boosting Machines  algorithms ( XGBoost) or Neural Networks can enhance the performance of the model .
Please assign me ,to work on this issue.

**Comment by BamaCharanChhandogi** [comment]
Thank you for creating this issue! 🎉 We'll look into it as soon as possible. In the meantime, please make sure to provide all the necessary details and context. If you have any questions or additional information, feel free to add them here. Your contributions are highly appreciated! 😊

You can also check our [CONTRIBUTING.md](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/blob/main/CONTRIBUTING.md) for guidelines on contributing to this project.

**Comment by BamaCharanChhandogi** [comment]
Don't need it at this time.
